portfolio/src/main/java/com/google/sps/servlets/EditDataServlet.java

```diff
@@ -43,72 +45,78 @@
-      // Get comment's id
-      Object input = comment.getKey().getId();
-      long commID = 0;
-      if (input instanceof Long) {
-        commID = (long) input;
-      }
-      else {
-        logger.warning("Could not convert comment's ID to long"); 
-      }
-
-      for (Entity user : userResults.asIterable()) {
-        ArrayList<Long> userLikes = null;
-        Object objUserLikes = user.getProperty("likes");
-        if (objUserLikes instanceof ArrayList) {
-            userLikes = (ArrayList<Long>) objUserLikes;
+      for (Entity comment : commResults.asIterable()) {
+        // Get comment's id
+        Object input = comment.getKey().getId();
+        long commID = 0;
+        if (input instanceof Long) {
+          commID = (long) input;
         }
         else {
-            logger.warning("Could not convert User's likes list to ArrayList<Long>"); 
+          logger.warning("Could not convert comment's ID to long"); 
         }
+  
+        for (Entity user : userResults.asIterable()) {
+          ArrayList<Long> userLikes = null;
+          Object objUserLikes = user.getProperty("likes");
+          if (objUserLikes instanceof ArrayList) {
+            userLikes = (ArrayList<Long>) objUserLikes;
+          }
+          else {
+            logger.warning("Could not convert User's likes list to ArrayList<Long>"); 
+          }
 
-        // check to make sure user's list of liked comments doesn't have this comment inside of it
-        if (!userLikes.contains(commID)) {
+          // check to make sure user's list of liked comments 
+          // doesn't have this comment inside of it
+          if (!userLikes.contains(commID)) {
           
-          if (type.equals("like")) {
-            Object likes = comment.getProperty("likes");
-            if (likes instanceof Long) {
-              comment.setProperty("likes", ((long) likes) + 1);
-            }
-            else {
-              logger.warning("Could not convert comment's likes to long"); 
-            }
-          }
-          else if (type.equals("dislike")) {
-            Object dislikes = comment.getProperty("dislikes");
-            if (dislikes instanceof Long) {
-              comment.setProperty("dislikes", ((long) dislikes) + 1);
+            if (type.equals("like")) {
+              Object likes = comment.getProperty("likes");
+              if (likes instanceof Long) {
+                comment.setProperty("likes", ((long) likes) + 1);
+              }
+              else {
+                logger.warning("Could not convert comment's likes to long"); 
+              }
             }
-            else {
-              logger.warning("Could not convert comment's dislikes to long"); 
+            else if (type.equals("dislike")) {
+              Object dislikes = comment.getProperty("dislikes");
+              if (dislikes instanceof Long) {
+                comment.setProperty("dislikes", ((long) dislikes) + 1);
+              }
+              else {
```

Comment:
No line break for `else`. https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style

Please try to use IntelliJ or a linter, which auto-fix these problems.

Reply:
Got it, thanks.